code/datums/status_effects/debuffs/debuffs.dm

```diff
@@ -202,10 +202,20 @@
 		if(locate(/obj/item/pillow) in owner.loc)
 			healing += 0.1
 
-		if(healing > 0 && health_ratio > 0.8)
-			owner.adjustBruteLoss(-1 * healing, required_bodytype = BODYTYPE_ORGANIC)
-			owner.adjustFireLoss(-1 * healing, required_bodytype = BODYTYPE_ORGANIC)
-			owner.adjustToxLoss(-1 * healing * 0.5, TRUE, TRUE, required_biotype = MOB_ORGANIC)
+		if(healing > 0)
+			// gives a boost to internal organs healing
+			for(var/obj/item/bodypart/target_bodypart in owner.bodyparts)
+				for(var/obj/item/organ/internal/target_organ in target_bodypart.contents)
```

Comment:
currently, organs are not stored inside bodyparts unless they are external organs.
this will only work after #76381 is merged, and even then it should be target_bodypart.organs

Reply:
Good to know.  I will wait until your PR gets merged before this continues.